Bottom: 828ce184223b4462fc4c8dec8378717a00cd2ac9
Top:    a8de699f6e4759dc6b118729796dbd6daf77b641
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-11 12:23:16 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 3b0b0d4..e29d62d 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -331,13 +331,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
